EnglishTranslation-master/QUEST_JOBSTEP.tsv

```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
 QUEST_JOBSTEP_20150730_003084	Talk to the Squire Master
 QUEST_JOBSTEP_20150730_003085	Collected all the pieces of Legwyn Family Seal. Return to the Squire Master.
 QUEST_JOBSTEP_20150730_003086	Search for the necklace of the great pirate, Bradislovas
-QUEST_JOBSTEP_20150730_003087	Corsair Master wants to look for the objects of the great pirate, Bradislovas and use them as a proof of the victory. Look for the objects of Bradislovas that went under the sea at 32_1 of Coral Plain which once was sea. 
+QUEST_JOBSTEP_20150730_003087	Corsair Master wants to look for the objects of the great pirate, Bradislovas, and use them as a proof of the victory. Look for Bradislovas's objects that went under the sea at 32_1 of Coral Plain which have since dried up. 
```

Comment:
*The* Corsair Master 
Bradislovas -> Vladslovas (can you mass replace this, please?)
of the great pirate, Bradislovas -> of the Great Pirate Vladslovas